src/utilities/collection-utils.test.ts

```diff
@@ -572,4 +572,124 @@ describe("CollectionUtils", () => {
     });
 
     // #endregion replaceElementAt
+
+    // -----------------------------------------------------------------------------------------
+    // #region sortByString
+    // -----------------------------------------------------------------------------------------
+
+    describe("#sortByString", () => {
+        type TestType = { letter: string };
+        const selector = (t: TestType) => t.letter;
+
+        it("Sorts an array by specified selector", () => {
```

Comment:
Nitpick, please start these tests statements with lowercase for consistency and being it is a part of a larger constructed sentence